client/my-sites/site-settings/settings-controller.js

```diff
@@ -93,7 +94,7 @@ export async function redirectGeneralSettingsIfDuplicatedViewsEnabled( context,
 		return page.redirect( `/sites/settings/site/${ siteSlug }` );
 	}
```

Comment:
I think this redirect is wrong, we should be updating the classic view menu item on the JP side instead of doing a redirect here. Will follow up.

Reply:
https://github.com/Automattic/wp-calypso/issues/99391